tests/src/image.rs

```diff
@@ -1,6 +1,8 @@
 use std::{borrow::Cow, ffi::OsStr, io, path::Path};
+
 use wgpu::util::DeviceExt;
 use wgpu::*;
+use wgt::math::align_to;
```

Comment:
We shouldn't dep on wgt if we can help it, this is re-exported in wgpu.

Reply:
fixed